### DISCUSSION TITLE
how to increase fontsize of any module ?

### DISCUSSION THREAD

**Discussion by fr05t-byt3** [polybar/polybar#2964] [Q&A] [answered]
![image](https://github.com/polybar/polybar/assets/74895351/10993acf-6825-4ac4-860c-b1000ec1b8e8)

[module/menu]
type = custom/script
exec = echo " "
click-left =  rofi -show drun
format-background = ${colors.primary}
format-foreground = ${colors.background}
format-padding = 5
font-size = 10px

**Comment by patrick96** [comment]
The font size is set per font (`font-0`, etc. in the bar section) and not per module.

**Reply by fr05t-byt3** [reply]
is there no way to increase size per module (some kinda work around) ?

**Reply by patrick96** [reply, marked as answer]
You could add another font with a bigger size and use that for the module with either the `-font` property on formats and labels or the font formatting tag
